lime/tools/helpers/IOSHelper.hx

```diff
@@ -319,144 +320,52 @@ class IOSHelper {
 	}
 	
 	
-	public static function launch (project:HXProject, workingDirectory:String):Void {
-		
-		initialize (project);
-		
+	public static function launch(project: HXProject, workingDirectory: String): Void {
+		initialize(project);
 		var configuration = "Release";
-			
 		if (project.debug) {
-			
 			configuration = "Debug";
-			
 		}
-		
-		if (project.targetFlags.exists ("simulator")) {
-			
+		if (project.targetFlags.exists("simulator")) {
 			var applicationPath = "";
-			
-			if (Path.extension (workingDirectory) == "app" || Path.extension (workingDirectory) == "ipa") {
-				
+			if (Path.extension(workingDirectory) == "app" || Path.extension(workingDirectory) == "ipa") {
 				applicationPath = workingDirectory;
-				
 			} else {
-				
 				applicationPath = workingDirectory + "/build/" + configuration + "-iphonesimulator/" + project.app.file + ".app";
-				
-			}
-			
-			var templatePaths = [ PathHelper.combine (PathHelper.getHaxelib (new Haxelib ("lime")), "templates") ].concat (project.templatePaths);
-			
-			var output = ProcessHelper.runProcess ("", "xcrun", [ "simctl", "list", "devices" ]);
-			var lines = output.split ("\n");
-			var foundSection = false;
-			
-			var device, deviceID;
-			var devices = new Map<String, String> ();
-			
-			var currentDeviceID = null;
-			
-			for (line in lines) {
-				
-				if (StringTools.startsWith (line, "--")) {
-					
-					if (line.indexOf ("iOS") > -1) {
-						
-						foundSection = true;
-						
-					} else if (foundSection) {
-						
-						break;
-						
-					}
-					
-				} else if (foundSection) {
-					
-					device = StringTools.trim (line);
-					device = device.substring (0, device.indexOf ("(") - 1);
-					device = device.toLowerCase ();
-					device = StringTools.replace (device, " ", "-");
-					
-					deviceID = line.substring (line.indexOf ("(") + 1, line.indexOf (")"));
-					
-					if (deviceID.indexOf ("inch") > -1) {
-						
-						var startIndex = line.indexOf (")") + 2;
-						deviceID = line.substring (line.indexOf ("(", startIndex) + 1, line.indexOf (")", startIndex));
-						
-					}
-					
-					devices.set (device, deviceID);
-					
-					if (project.targetFlags.exists (device)) {
-						
-						currentDeviceID = deviceID;
-						break;
-						
-					}
-					
-				}
-				
-			}
-			
-			if (currentDeviceID == null) {
-				
-				if (project.targetFlags.exists ("ipad")) {
-					
-					currentDeviceID = devices.get ("ipad-air");
-					
-				} else {
-					
-					currentDeviceID = devices.get ("iphone-6");
-					
-				}
-				
 			}
+			// TODO: check if needed as not used
+			var templatePaths = [PathHelper.combine(PathHelper.getHaxelib(new Haxelib("lime")), "templates")].concat(project.templatePaths);
```

Comment:
this prop is not used in the code in this branch safe to remove or it should be used?